srcpkgs/golang-migrate/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'golang-migrate'
+pkgname=golang-migrate
+version=4.14.1
+revision=1
+wrksrc="migrate-${version}"
+build_style=go
+go_import_path="github.com/golang-migrate/migrate/v4/cmd/migrate"
```

Comment:
```suggestion
go_import_path="github.com/golang-migrate/migrate/v4"
go_package="cmd/migrate"
```

I think. Even if the value isn't being used anyway :p

Can probably omit `go_package`.

Reply:
Ok, removed the trailing `cmd/migrate`.  I didn't include `go_package` since (as you said) the value isn't being used.